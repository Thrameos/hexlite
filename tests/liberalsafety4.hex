% This program is domain-expansion safe because Z is bounded due to #maxint

#maxint=30.
q(1).
q(9).
p(1).
p(Z) :- p(S), &testConcat[S,0](Z), Z=#sum{X:q(X)}.
