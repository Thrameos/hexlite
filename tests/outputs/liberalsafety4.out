{q(1),q(9),p(1),p(10)}
